### PR TITLE
feat: remove references to CCD UI from FPL when live(FPLA-1772)

### PR DESCRIPTION
**Pull request by KiranVarma02**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/FPLA-1772

### Change description ###

Remove references to CCD UI from FPL when live

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ *] No
```

### CODE REVIEW

File: k8s/aat/common/family-public-law/fpl-case-service.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ spec:
   chart:
     repository: https://hmctspublic.azurecr.io/helm/v1/repo/
     name: fpl-case-service
-    version: 1.11.4
+    version: 1.11.7
```
Comment (lewisbirks) [inline k8s/aat/common/family-public-law/fpl-case-service.yaml:18]: should this not be .8? same in others as well

Comment (KiranVarma02) [reply k8s/aat/common/family-public-law/fpl-case-service.yaml:18]: oh, its moved to 8 now..it was 7 when i created PR :)

Comment (KiranVarma02) [reply k8s/aat/common/family-public-law/fpl-case-service.yaml:18]: This PR is not going to be merged until after expert UI is live. So i guess i will make this change later.

Comment (tomaszpowroznik) [reply k8s/aat/common/family-public-law/fpl-case-service.yaml:18]: it basically should match version in your fpl-ccd-configuration Chart.yaml which currently is .8

**Pull request merged**